Solutions/Digital Shadows/Data Connectors/Digital Shadows/DigitalShadowsSearchlight_API_functionApp.json

```diff
@@ -0,0 +1,147 @@
+{
+    "id": "DigitalShadowsSearchlightAzureFunctions",
+    "title": "Digital Shadows Searchlight",
+    "publisher": "Digital Shadows",
+    "descriptionMarkdown": "The Digital Shadows data connector provides ingestion of the incidents and alerts from Digital Shadows Searchlight into the Azure Sentinel using the REST API. The connector will provide the incidents and alerts information such that it helps to examine, diagnose and analyse the potential security risks and threats.",
```

Comment:
Hey @shikhin-metron, also please change Azure Sentinel to Microsoft Sentinel, thanks.